src/module.js

```diff
@@ -1,7 +1,13 @@
 /**
  * @constructor
  */
-Sk.builtin.module = function module () {
+Sk.builtin.module = function module (name) {
+    if (!(this instanceof Sk.builtin.module)) {
+        return new Sk.builtin.module(name);
+    }
+    this["$d"] = {__name__: name};
+    this["$d"]["__dict__"] = this["$d"];
+    return this;
 };
```

Comment:
Should we have the `__dict__` here? this would return a javascript object to the user no?

Reply:
I totally defer to you on this one, I figured whatever I had in here would have to be heavily changed for your new PRs anyway. This is just what I had at this point in time. I note that my own local Skulpt (which integrates most of your changes, but not the latest ones) actually looks like this:

```javascript
Sk.builtin.module = Sk.abstr.buildNativeClass("module", {
// ...
        tp$init: function(args, kwargs) {
            Sk.abstr.checkArgsLen(this.tp$name, args, 1, 3);
            this["$d"] = {
                "__name__": args[0],
                "__package__": Sk.builtin.none.none$,
            };
            return Sk.builtin.none.none$;
        },
```

Reply:
Nice. Yeah, For this pr I'd remove the `__dict__` since the user won't get a python object if they try to access it. I guess this really should be a python dictionary in the future...

Reply:
Does b92c363d246cd672250208165c4ad4a3ac9fbae4 handle this? I feel silly for letting my module stuff slip into this PR, it's not really all that relevant, ultimately.